test/lib/mayaUsd/render/mayaToHydra/cpp/testUtils.h

```diff
@@ -144,6 +144,27 @@ class SceneIndexInspector
     HdSceneIndexBasePtr _sceneIndex;
 };
 
+class PrimNamePredicate
+{
+public:
+    PrimNamePredicate(const std::string& primName) : _primName(primName) {}
+
+    /**
+     * @brief Predicate to match a prim name.
+     *
+     * @param[in] sceneIndex The scene index in which the prim path to test is contained.
+     * @param[in] primPath The prim path to test.
+     *
+     * @return True if the argument prim path's name matches the predicate's prim name, false otherwise.
+     */
+    bool operator()(const HdSceneIndexBasePtr& sceneIndex, const SdfPath& primPath) {
```

Comment:
sceneIndex argument is unused?

Reply:
Yes, as this class is to be used as a FindPrimPredicate, whose signature includes a SceneIndex. This was indeed not explained, so I've renamed the unused scene index variable to `_`, mentioned that it is unused and clarified why that is.